source/common/sequencer_impl.cc

```diff
@@ -158,6 +157,8 @@ void SequencerImpl::waitForCompletion() {
   ASSERT(!running_);
 }
 
+void SequencerImpl::cancel() { cancelled_ = true; }
```

Comment:
It seems like this PR has a partial implementation of request cancellation.. should this just be added as a supported request type or should these be removed?